server/controllers/stock/reports/stock_adjustment.receipt.handlebars

```diff
@@ -126,7 +126,7 @@
             <td class="text-right">{{old_quantity}}</td>
             <td class="text-right">{{new_quantity}}</td>
             <td class="text-right">{{difference}}</td>
-            <td class="text-right">{{currency unit_cost ../enterprise.currency_id}}</td>
+            <td class="text-right">{{currency unit_cost ../enterprise.currency_id 4}}</td>
```

Comment:
Question: Do you need to insert the '4' here now that you're forcing 4 if the number is too small?

Reply:
Indeed, you will also notice that the option to display four decimals has only been used in very specific cases in cases where one seeks to display the unit cost of an inventory I had not thought generalized this rule throughout the system unless deemed necessary to apply everywhere.

Reply:
Sounds good!